src/main/webapp/scripts/register.js

```diff
@@ -0,0 +1,34 @@
+let url = "https://translation.googleapis.com/language/translate/v2/languages"
```

Comment:
What does this file do?

Reply:
I was originally not going to use the Java Client and was gonna make direct calls to the endpoint with fetch, and then realized it would be much easier to do it the other way. Removed these code remnants.